spec/modules/scorecard_api/service_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+describe ScorecardApi::Service do
```

Comment:
This also would be a good use case for a VCR cassette so that you aren't mocking responses.
And you can check that `.populate` is returning an Array of Scorecard objects, etc.

Reply:
add check that all objects are Scorecard objects